addon/utils/serializer-partial-model-handler.js

```diff
@@ -0,0 +1,33 @@
+export default {
+  preNormalize: function(modelClass, resourceHash, context, normalizePartialRelationshipStrategy) {
```

Comment:
I would call this something like normalizeEachPartialRelationship which is it's role, rather then by position which is before normalize. When you see the code, you can tell it's called before normalize, so the name doesn't really help you